aQute.libg/test/aQute/lib/io/IOTest.java

```diff
@@ -297,8 +322,10 @@ public void testCreateSymlinkOrCopyWillDeleteOriginalLink() throws Exception {
 		}
 	}
 
-	public void testCreateDirectory_Symlink() throws Exception {
-		Path rootDirectory = Paths.get("generated/tmp/test/" + getName());
+	@Test
+	@DisabledOnOs(WINDOWS)
+	public void testCreateDirectory_Symlink(TestInfo info) throws Exception {
+		Path rootDirectory = Paths.get("generated/tmp/test/" + info.getTestMethod());
```

Comment:
Also, this is wrong since `getTestMethod()` returns `Optional<Method>` and not a string. I think you need `info.getTestMethod().map(Method::getName).get()`

Reply:
```java
	private String	testName;
	@BeforeEach
	public void before(TestInfo info) {
		testName = info.getTestMethod()
			.map(Method::getName)
			.get();
	}
```

Reply:
I had falsely assumed that Method.toString() would be a friendly String of the method name but it's not that friendly at all. I've used getDisplayName() which prints as `testCreateDirectory_Symlink(TestInfo)` which should be good enough for a temp directory.

Reply:
Changed it again - looked more closely at the second post there and saw that was better still & I've implemented that.